docs/apis/data_en.md

```diff
@@ -0,0 +1,205 @@
+
+The argument list of `decode_image()` function is as follows:
+
+|Parameter Name|Type|Parameter Description|Default Value|
+|-------|----|--------|-----|
+|`im_path`|`str`|Path of input image.||
+|`to_rgb`|`bool`|If `True`, the conversion of BGR to RGB format is performed. This parameter is not used and may be removed in the future. Do not use it if possible.|`True`|
+|`to_uint8`|`bool`|If `True`, the image data read is quantized and converted to uint8.|`True`|
+|`decode_bgr`|`bool`|If `True`, automatically parses non-geo format images (such as jpeg images) into BGR format.|`True`|
+|`decode_sar`|`bool`|If `True`, single-channel geo-format images (such as GeoTiff images) are automatically parsed as SAR images.|`True`|
+|`read_geo_info`|`bool`|If `True`, the geographic information is read from the image.|`False`|
+|`use_stretch`|`bool`|Whether to stretch the image brightness by 2% linear. Take effect only if `to_uint8` is `True`.|`False`|
+|`read_raw`|`bool`|If `True`, it is equivalent to specifying `to_rgb` as `True` and `to_uint8` as `False`, and this parameter has a higher priority than the above.|`False`|
+
+The return format is as follows:
+
+- If `read_geo_info` is `False`, the image data ([h, w, c] arrangement) is returned in the format of np.ndarray.
+- If `read_geo_info` is `True`, a binary group is returned, in which the first element is the image data read, the second element is a dictionary, in which the key-value pair is the geographic information of the image, such as geographic transformation information, geographic projection information, etc.
+
+## Data Transformation Operator
+
+In PaddleRS a series of classes are defined that, when instantiated, perform certain data preprocessing or data augmentation operations by calling the `__call__` method. PaddleRS calls these classes data preprocessing/data augmentation operators, and collectively **Data Transform Operators**. All data transformation operators inherit from the parent class[`Transform`](https://github.com/PaddlePaddle/PaddleRS/blob/develop/paddlers/transforms/operators.py).
+
+### `Transform`
+
+The `__call__` method of the `Transform` object takes a unique argument `sample`. `sample` must be a dictionary or a sequence of dictionaries. When `sample` is a sequence, perform data transformations for each dictionary in `sample` and return the results sequentially stored in a Python build-in list; When `sample` is a dictionary, the `Transform` object extracts input from some of its key-value pairs (these keys are called "input keys"), performs the transformation, and writes the results as key-value pairs into `sample`(these keys are called "output keys"). It should be noted that many of the `Transform` objects in PaddleRS currently have a carbon copy behavior, that is, an intersection between the input key and the output key. The common key names in `sample` and their meanings are as follows:
```

Comment:
It should be noted that many of the `Transform` objects in PaddleRS overwrite key-value pairs, that is, there is an intersection between the input key and the output key. -> ``